src/script/util/Environment.ts

```diff
@@ -62,11 +53,11 @@ export const Environment = {
     }
 
     if (doNotFormat) {
-      return getAppVersion();
+      return Config.getConfig().VERSION;
```

Comment:
There is no formatting anymore so we should only check in this function if we want to show the electron version or not.